lib/shelley/src/Cardano/Wallet/Shelley/Pools.hs

```diff
@@ -207,7 +219,42 @@ newStakePoolLayer
             & view #retirement
             & fmap (view (#epochNumber . #getApiT))
 
-    getTip = liftIO $ unsafeRunExceptT currentNodeTip
+        -- Sort by non-myopic member rewards, making sure to also randomly sort
+        -- pools that have equal rewards (hence the IO).
+        --
+        -- NOTE: we discard the final value of the random generator because we
+        -- do actually want the order to be stable between two identical
+        -- requests. The order simply needs to be different between _wallets_.
```

Comment:
The pool db, and thus the seed is shared for all wallets on a single wallet installation, so `_wallets_` should be something else, right?

Reply:
Right, it's actually _server_. The server will return a consistent ordering (which is what we probably want to not be really confusing). Yet, between different user machines, the seed will be different.